2021/scoring/clean_result_csvs.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+SCRIPTDIR=`dirname $(readlink -f "$0")`
+
+PROCESS_CSV="$SCRIPTDIR/../../tools/process-results/process_results.py"
+RESULT_DIR="$SCRIPTDIR/../results"
+NONINC_DECISION="$SCRIPTDIR/../results/sq-disagreements-decision.csv"
+
+TMPDIR=$(mktemp -d)
+trap "rm -rf $TMPDIR" EXIT
+
+cat blacklist_incremental_*.txt > excluded_incremental.txt
+cat blacklist_nonincremental_*.txt > excluded_nonincremental.txt
```

Comment:
Where do the `blacklist_incremental_*.txt` and `blacklist_nonincremental_*.txt` come from?